.travis.yml

```diff
@@ -41,8 +41,8 @@ before_script:
 
 script:
     - phpunit --coverage-text
-    - ./vendor/symfony-cmf/testing/bin/server &
-    - ./vendor/bin/behat
+    - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then ./vendor/symfony-cmf/testing/bin/server & fi
+    - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then ./vendor/bin/behat; fi
```

Comment:
why not combining them? `if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then ./vendor/symfony-cmf/testing/bin/server; ./vendor/bin/behat; fi`